src/image_artifact.js

```diff
@@ -20,11 +20,11 @@ const resolveArtifactName = (imageName) => `action_image_artifact_${resolvePacka
  *      `image_artifact_foo_latest`. In short we will have an artifact like,
  *             image_artifact_foo_latest[foo_latest]
  */
-exports.upload = async function(image) {
+exports.upload = async function(image, retentionDays) {
```

Comment:
looks about right. Could u please update jsdoc ?

```suggestion
/**
 * @param {string} image 
 * @param {number} retentionDays 
 *
 * @returns {string} // Uploaded artifact name
 * 
 * Upload a image package as github artifact which is later possible to point to with given image name. 
 * Eg. image "foo:latest" is packaged to a file `foo_latest` and then upload as an artifact with a name 
 *      `image_artifact_foo_latest`. In short we will have an artifact like,
 *             image_artifact_foo_latest[foo_latest]
 */
exports.upload = async function(image, retentionDays) {
```

Reply:
now i think we could make this change backward compatible so that no major version needs to be bumped. What do u say if we have a default value for the `retentionDays` ?

And plz update minor version in `package.json` accordingly.

Reply:
> now i think we could make this change backward compatible so that no major version needs to be bumped. What do u say if we have a default value for the `retentionDays` ?

Ah..good catch. Updated in the current version.
 
> And plz update minor version in `package.json` accordingly.

Done.